netbox_inventory/views/inventoryitem_group.py

```diff
@@ -49,6 +49,40 @@ def get_extra_context(self, request, instance):
 
         # get counts for each inventoryitem type and status combination
         type_status_counts = asset_counts_type_status(instance, assets)
+
+        # change structure of stored objects
+        type_status_objects = []
+        prev_type = 0
+        asset_obj = {}
+        total_items = len(type_status_counts)-1 # get last index
+        for idx, tsc in enumerate(type_status_counts):
+            if prev_type != tsc['inventoryitem_type']:
+                prev_type = tsc['inventoryitem_type']
+                # make sure skip first insert
+                if len(asset_obj.keys()) != 0:
+                    type_status_objects.append(asset_obj)
+                asset_obj = {}
+
+            # collection of statuses for same types
+            status_list = { 
+                'status': tsc['status'],
+                'count': str(tsc['count']), # needs to be a string to render
+                'color': tsc['color']
```

Comment:
```suggestion
                'color': tsc['color'],
                'label': tsc['label']
```

Add status label (aka display)